app/components/web-components/vl-modal-footer.hbs

```diff
@@ -1,32 +1,32 @@
-{{!-- TODO: octane-refactor --}}
-{{!-- template-lint-disable no-action --}}
-<Auk::Toolbar @size="large">
-  <Auk::Toolbar::Group @position="left">
-    <Auk::Toolbar::Item>
-      <AuButton
-        data-test-vl-modal-footer-cancel
-        @skin="naked"
-        {{action "cancelAction"}}
-      >
-        {{this.cancelButtonText}}
-      </AuButton>
-    </Auk::Toolbar::Item>
-  </Auk::Toolbar::Group>
-  <Auk::Toolbar::Group @position="right">
-    <Auk::Toolbar::Item>
-      <AuButton
-        data-test-vl-modal-footer-save={{true}}
-        @skin="primary"
-        {{!-- also disable button when 'isLoading', since no loading state on button is available.
-          Disabling the button while in loading state is important in order to avoid multiple
-          action calls when clicking multiple times on the same confirm-button --}}
-        @disabled={{or this.disableSave this.isLoading}}
-        {{!-- @loading={{this.isLoading}}
-        @loadingMessage={{t "please-be-patient"}} --}}
-        {{action "saveAction"}}
-      >
-        {{this.saveButtonText}}
-      </AuButton>
-    </Auk::Toolbar::Item>
-  </Auk::Toolbar::Group>
-</Auk::Toolbar>
\ No newline at end of file
+<div class="auk-modal__footer auk-modal__footer--bordered">
+  <Auk::Toolbar @size="large">
+    <Auk::Toolbar::Group @position="left">
+      <Auk::Toolbar::Item>
+        <AuButton
+          data-test-vl-modal-footer-cancel
+          @skin="naked"
+          {{on "click" @cancelAction}}
+        >
+          {{@cancelButtonText}}
+        </AuButton>
+      </Auk::Toolbar::Item>
+    </Auk::Toolbar::Group>
+    <Auk::Toolbar::Group @position="right">
+      <Auk::Toolbar::Item>
+        <AuButton
+          data-test-vl-modal-footer-save={{true}}
+          @skin="primary"
+          {{!-- also disable button when 'isLoading', since no loading state on button is available.
```

Comment:
Does this comment still hold? `AuButton` has an `@loading` argument now.

Reply:
Yes and no. There's the `@loading` argument now which we can use, but setting it doesn't disable the button, so a user could click it multiple times while it's still loading.